bindings/java/src/org/sleuthkit/datamodel/TskEvent.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Sleuth Kit Data Model
+ * 
+ * Copyright 2020 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.datamodel;
+
+/**
+ *
+ * Defines an interface implemented by events published by Sleuthkit.
+ */
+public interface TskEvent {
```

Comment:
I'd propose we name this something like TskEventWithDataSource since we may have events for reports, which are not always tied to a single DS.